### PR TITLE
improvement(results): store cycle's 'start time'

### PR DESCRIPTION
**Pull request by soyacz**
### Testing

- [x] - [example result](https://argus.scylladb.com/test/47ee4c1d-bff9-42fc-8911-29bc79f2ef40/runs?additionalRuns[]=9684194c-1815-4dd9-a717-f82300d001cc) - see final solution moved 'start time' column before screenshot links.

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

**Review by juliayakovlev — DISMISSED** [review]
LGTM
What about end time - do we want it?

**Comment by soyacz** [comment]
> LGTM What about end time - do we want it?

there's duration shown, would be bit redundant I think

File: sdcm/argus_results.py [hunk 1]
```diff
@@ -41,8 +43,9 @@ class Meta:
             ColumnMetadata(name="Throughput write", unit="op/s", type=ResultType.INTEGER, higher_is_better=True),
             ColumnMetadata(name="Throughput read", unit="op/s", type=ResultType.INTEGER, higher_is_better=True),
             ColumnMetadata(name="duration", unit="HH:MM:SS", type=ResultType.DURATION, higher_is_better=False),
+            ColumnMetadata(name="start time", unit="", type=ResultType.TEXT),
```
Comment (fruch) [inline sdcm/argus_results.py:46]: Why don't we want this as a number ?

Do we assume we'll never plot or sort by this one ?

File: utils/migrate_latency_results_to_argus.py [hunk 2]
```diff
@@ -81,13 +81,17 @@ def sort_func(item):
...
                     send_result_to_argus(argus_client=client, workload=workload, name=operation,
-                                         description=description, cycle=0, result=result)
+                                         description=description, cycle=0, result=result, start_time=start_time)
                 except argus.client.base.ArgusClientError:
                     print(
                         f"Failed to send {operation} - {workload} - latencies to Argus: {hit['_source']['test_details']['job_url']}")
                 continue
             for idx, cycle in enumerate(latency_during_ops[operation]["cycles"], start=1):
+                start_time = cycle["hdr_summary"].get("READ", cycle["hdr_summary"].get("WRITE"))["start_time"]
+                start_time = start_time if start_time < 1000000000000 else start_time / 1000
```
Comment (fruch) [inline utils/migrate_latency_results_to_argus.py:94]: What is this magic number?

Comment (soyacz) [reply utils/migrate_latency_results_to_argus.py:94]: for unknown reasons, sometimes we get timestamp in milliseconds and sometimes in seconds

File: sdcm/argus_results.py (hunk 1 above)
Comment (soyacz) [reply sdcm/argus_results.py:46]: no, it just serves investigation purposes to quickly get to proper log place (or looking at monitoring).

Comment (fruch) [reply sdcm/argus_results.py:46]: maybe we should document this at some place in the code ?

File: utils/migrate_latency_results_to_argus.py (hunk 2 above)
Comment (soyacz) [reply utils/migrate_latency_results_to_argus.py:94]: one saved in ES - usually Steady State (but i think I spotted also in some other cases - hard to tell).

File: sdcm/argus_results.py (hunk 1 above)
Comment (soyacz) [reply sdcm/argus_results.py:47]: I added a comment line. Also found that rebasing have 'eaten' changes in other tables - fixed.

**Review by fruch — APPROVED** [review]

**Pull request merged**